src/js/index.js

```diff
@@ -448,64 +465,77 @@ export const generate = (baseSpacing = 24, scale = 6) => {
             radius: `${baseSpacing * 0.25}px`, // 4px
           },
           pad: {
-            vertical: `${baseSpacing / 2.285}px`,
-            horizontal: `${baseSpacing / 1.6}px`,
+            vertical: `${baseSpacing * 0.125}px`,
+            horizontal: `${baseSpacing * 0.5}px`,
           },
         },
         medium: {
           border: {
-            radius: `${baseSpacing * 0.312}px`, // 5px
+            radius: `${baseSpacing * 0.25}px`,
           },
           pad: {
-            vertical: `${baseSpacing / 1.618 - borderWidth}px`,
-            horizontal: `${baseSpacing - borderWidth * 1.4}px`,
+            vertical: `${baseSpacing * 0.375}px`,
+            horizontal: `${baseSpacing}px`,
           },
         },
         large: {
           border: {
-            radius: `${baseSpacing * 0.4}px`, // 24px
+            radius: `${baseSpacing * 0.25}px`,
+          },
+          pad: {
+            vertical: `${baseSpacing * 0.5}px`,
+            horizontal: `${baseSpacing * 1.5}px`,
           },
         },
       },
       border: {
         // color: { dark: undefined, light: undefined }
         width: `${borderWidth}px`,
-        radius: `${baseSpacing * 0.312}px`,
+        radius: `${baseSpacing * 0.25}px`,
       },
       color: { dark: undefined, light: undefined },
       default: {
         background: 'transparent',
         border: 'none',
-        // color: undefined,
-        padding: {
-          vertical: '0',
-          horizontal: '0',
+        color: 'dark-7',
+        extend: {
+          fontWeight: 'bold',
+          textTransform: 'uppercase',
```

Comment:
Why there are so many extends, can it be achieved without using extend, ike using elevation instead of box-shadow